src/gazebo_wind_plugin.cpp

```diff
@@ -134,13 +132,19 @@ void GazeboWindPlugin::OnUpdate(const common::UpdateInfo& _info) {
   }
 
   gazebo::msgs::Vector3d* force = new gazebo::msgs::Vector3d();
-  force->set_x(wind.X() + wind_gust.X());
-  force->set_y(wind.Y() + wind_gust.Y());
-  force->set_z(wind.Z() + wind_gust.Z());
+  force->set_x( wind_gust.X());
+  force->set_y( wind_gust.Y());
+  force->set_z( wind_gust.Z());
+
+  gazebo::msgs::Vector3d* wind_v = new gazebo::msgs::Vector3d();
+  wind_v->set_x( wind.X());
+  wind_v->set_y( wind.Y());
+  wind_v->set_z( wind.Z());
```

Comment:
```suggestion
  wind_v->set_z(wind.Z());
```
Why's there always a space.